package.json

```diff
@@ -75,13 +75,14 @@
     "@sendgrid/mail": "^6.2.1",
     "@vimeo/player": "^2.2.0",
     "algoliasearch": "^3.13.1",
+    "animejs": "^3.0.1",
     "async": "2.5.0",
     "aws-sdk": "~2.0.0",
     "backbone": "~1.1.1",
     "basic-auth": "^1.0.4",
     "bayesian-battle": "0.0.7",
     "bitly": "^5.0.12",
-    "bluebird": "^3.2.1",
+    "bluebird": "^3.5.4",
```

Comment:
What are the advantages of upgrading this?

Reply:
The upgrade is non breaking and includes bug fixes. Bluebird is required for Promises that are used in Commands so it  feels safer to use a more updated one.